src/style.css

```diff
@@ -1,3 +1,99 @@
-body {
+* {
   margin: 0;
+  padding: 0;
```

Comment:
it is redundant to reset margin and padding for all selectors. reset only for elements which really need it